Packs/MicrosoftGraphDeviceManagement/ReleaseNotes/1_1_24.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Microsoft Endpoint Manager (Intune)
+
+Updated the Documentation to be more accurate.
```

Comment:
Do we need to be more specific?

- Updated the *page_size* argument description.